theano/tensor/tests/test_blas.py

```diff
@@ -974,7 +979,7 @@ def cmp(a_shp, b_shp, c_shp, sqr_shp=(5, 5)):
                                                 c * a * cst * T.dot(a, b),
                                                 mode=m2)
                             topo = f.maker.fgraph.toposort()
-                            check_dot22scalar(f, 2)
+                            check_dot22scalar(f, 5)
```

Comment:
Considering this test is now used for gemm_inplace and not _dot_22scalar, this function and the test should probably be renamed to correspond.